awscrt/http.py

```diff
@@ -532,6 +532,20 @@ def __str__(self):
         return self.__class__.__name__ + "(" + str([pair for pair in self]) + ")"
 
 
+class HttpProxyConnectionType(IntEnum):
+    """Proxy connection type enumeration"""
+    Legacy = 0
+    """Use the old connection establishment logic that would use
+         (1) Forwarding if not using tls
+         (2) Tunneling if using tls"""
```

Comment:
Update the HTML docs by running `./make-docs.sh`
Look at the result by opening `docs/index.html` to see if the formatting looks good

you might need to do like:
```
1. Forward ifblahbalh
2. Tunneling if blahlbalh
```
check the [docs](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks) if that doesn't work

Reply:
I'm getting the following:

reading sources... [100%] index                                                                                                            

Sphinx error:
master file /home/bret/projects/crt/aws-crt-python/docsrc/source/contents.rst not found